Dockerfile

```diff
@@ -39,7 +39,7 @@ LABEL maintainer="IBM Quality Measure and Cohort Service Team" \
       description="Quality Measure and Cohort Service" \
       name="cohorting-app" \
       vendor="Alvearie Open Source by IBM" \
-      version="1.0" \
+      version="2.0.1" \
       release="1" \
```

Comment:
Should this say release="2"? I'm not really clear on how that field works, but it feels like that is the major number from the semantic version. Or, do we even need that release tag? The [IBM FHIR team](https://github.com/IBM/FHIR/blob/1e8def81b69261b77ba175e3ec5e247a0b2a6cec/fhir-install/Dockerfile#L31) doesn't use it. This also begs the question about whether or not we need these tags in our other Dockerfiles like cohort-evaluator-spark/Dockerfile and docker/spark-history-server/Dockerfile.

Reply:
These tags are optional in the dockerfiles, but I think are used if we want to create certified Red Hat images, which we don't have a requirement to do. Last release we didn't even update it at all. If someone tries to get the information on the docker image, I think is the only place this info shows up. I can change to release 2